### PR TITLE
Fix customer og urls

### PR DESCRIPTION
**Pull request by fsansalvadore**
## Before 

Screenshot 2023-06-13 at 11 55 41

## After

Screenshot 2023-06-13 at 11 56 59

### CODE REVIEW

File: apps/www/pages/customers/[slug].tsx [hunk 1]
```diff
@@ -5,8 +5,7 @@ import { MDXRemote } from 'next-mdx-remote'
 import { NextSeo } from 'next-seo'
 import Image from 'next/image'
 import Link from 'next/link'
-import { useRouter } from 'next/router'
-import { IconChevronLeft, IconChevronsLeft } from '~/../../packages/ui'
+import { IconChevronLeft } from '~/../../packages/ui'
```
Comment (alaister) [inline apps/www/pages/customers/[slug].tsx:8]: Can this import just be `import { IconChevronLeft } from 'ui'`?

File: apps/www/pages/customers/[slug].tsx [hunk 2]
```diff
@@ -59,13 +58,14 @@ export async function getStaticProps({ params }: any) {
...
   const content = props.blog.content
-  const { basePath } = useRouter()
 
   const meta = {
     title: props.blog.meta_title ?? `${props.blog.name} | Supabase Customer Stories`,
     description: props.blog.meta_description ?? props.blog.description,
-    image: props.blog.og_image ?? `${basePath}/images/customers/og/customer-stories.jpg`,
-    url: `https://supabase.io/customers/${props.blog.slug}`,
+    image:
+      `https://supabase.com${props.blog.og_image}` ??
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/pages/customers/[slug].tsx:66]: for site URL that is

[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/pages/customers/[slug].tsx]: true 👍

**Review by MildTomato — APPROVED** [review]

**Review by alaister — APPROVED** [review]

**Comment by MildTomato** [comment]
🚀 🚀 🚀

**Pull request merged**